packages/cloud-cognitive/src/components/Datagrid/useNestedRows.js

```diff
@@ -32,12 +32,10 @@ const useNestedRows = (hooks) => {
       style: {
         marginLeft: `${row.depth > 0 ? marginLeft : 0}px`,
```

Comment:
It looks like we need to remove this margin too in v11?

Seeing extra spacing and text misalignment here.
![image](https://user-images.githubusercontent.com/1717398/216699076-e0e3ea04-6664-4a37-93d2-569cfcc7b824.png)

Reply:
Margin removed.